docs/_docs/media-center-demo/index.md

```diff
@@ -220,10 +198,10 @@ app][17].
 Update the Media Center app to load your copy of the Figma Design:
 
 1.  Open the app's `MainActivity.kt` file, located within the Android project at
-    `MediaCompose/mediacompose/java/com/android/designcompose/reference/mediacompose/MainActivity.kt`.
+    `DC Unbundled Apps/mediacompose/java/com/android/designcompose/reference/mediacompose/MainActivity.kt`.
```

Comment:
Think this should be `reference-apps/aaos-unbundled/mediacompose/src/main/java/com/android/designcompose/reference/mediacompose/MainActivity.kt`

Reply:
This line is referring to the location of the file within the Android view in Android Studio. The fild location is mentioned on the next line.